NOTICE

```diff
@@ -109,3 +109,8 @@ License - https://github.com/hexops/gotextdiff/blob/main/LICENSE
 https://github.com/BurntSushi/toml
 Copyright (c) 2013 TOML authors
 https://github.com/BurntSushi/toml/blob/master/COPYING
+
+dario.cat/mergo
+Copyright (c) 2013 Dario Castañé. All rights reserved.
+Copyright (c) 2012 The Go Authors. All rights reserved.
+https://github.com/darccio/mergo/blob/master/LICENSE
```

Comment:
Historical tidbit: we used to depend on this bundle configuration merging before the move to `libs/dyn`.

Reply:
Thanks. It looks solid. From the past experience, are there any issues or quirks to be aware of?

Reply:
No, it did its job well.